### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by simlay** [RibirX/Ribir#530]
**Is your feature request related to a problem? Please describe.**
This project is interesting. It's actually not far from basic iOS support. Commenting out the `run_on_demand` portions results in the project building for iOS. Hard to verify it's actually working without the main runloop though.

**Describe the solution you'd like**
That's hard to answer. If this project just wants to render the UI and have touch events propagate, moving away [away from `run_on_demand`](https://github.com/RibirX/Ribir/blob/855f3a1020952dc750d3f1b601c8b204057b9e1d/ribir/src/app.rs#L112-L114) as it [is not supported in iOS](https://github.com/rust-windowing/winit/blob/385c4b3c880cbd2b992f45b2d8fbe36000a3d30b/src/platform/mod.rs#L24-L32).

Getting virtual keyboard input I'm pretty sure depends on https://github.com/rust-windowing/winit/issues/2260

**Comment by mrgzi** [comment]
I agree. This project looks very interesting. But I can't find much information about the project. At least it would be nice to see a roadmap and also a Discord server.

I also create a topic about it in discussions: https://github.com/RibirX/Ribir/discussions/529

**Comment by wjian23** [comment]
@simlay Thank you for your attention. yeah, we will tack the feature related to the issue.
it's our goal to support all platform, and we have only test and verify it on desktop yeah, we will replenish a roadmap later. 1 Ribir is aim to construction the ui and react to the data change( which be trigger by event like touch, keyboard input and so on), So the run_on_demand can be replace to event_loop.run as well.  2 The issue https://github.com/rust-windowing/winit/issues/2260， we have not encounter yeah, I'll learn it later.

**Comment by wjian23** [comment]
> I agree. This project looks very interesting. But I can't find much information about the project. At least it would be nice to see a roadmap and also a Discord server.
> 
> I also create a topic about it in discussions: #529

Thank you @mrgzi , we will replenish a roadmap later. we'll prioritize support for wasm in about 2-3 month, and the mobile will be supported in about 4-6 month.  And will be very happy to have you join us to improve Ribir.
